setup.py

```diff
@@ -31,6 +31,7 @@
     url="https://github.com/mottosso/Qt",
     license="MIT",
     zip_safe=False,
+    data_files=['LICENSE'],
```

Comment:
Nitpick, please use consistent quotation marks `"` :)

Other than that, all good!

Reply:
Whoops. Good catch. Fixed.